docs/UserGuide.md

```diff
@@ -72,12 +72,53 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
+### Finding a person by fields: `find`
+
+Finds persons whose fields match the given keywords.
+
+Format: `find [n/KEYWORD …] [a/KEYWORD …] [e/KEYWORD …] [p/KEYWORD …] `
+
+* There should be at least one field i.e. `find` is an invalid command.
+* The search is case-insensitive. e.g`hans` will match `Hans`
+* The order of the keywords does not matter. e.g `Clementi West` will match `West Clementi`
+* The keyword does not need to match the field exactly. e.g. `john@gmail.com` will match `leejohn@gmail.com`
+* A field just needs to match at least one keyword. e.g. `Hans Bo` will match `Hans Gruber`, `Bo Yang`
+* All fields specified need to match for person to be matched.<br>
```

Comment:
Some bulleted sentences end with a period while others do not. In DG i standardised it to no periods for listed points, maybe we could make that a standard for both UG and DG

Reply:
I think if it's a sentence then should have fullstop

Reply:
Oic, I think you're right. I found an article about it [here](https://www.grammarly.com/blog/bullet-points/)
I'll need to double check the DG. I noticed that manyof the lists in the original AB3 don't have full stops too so we might need to double check that.